packages/shared/src/components/filters/TagsFilter.spec.tsx

```diff
@@ -63,33 +67,44 @@ const createAllTagCategoriesMock = (
 
 let client: QueryClient;
 
+const defaultAlerts: Alerts = { filter: false };
+const defaultFlags: IFlags = {};
+const myFeedOnEnabledFlag = { my_feed_on: { enabled: true, value: 'true' } };
```

Comment:
It's best to test this without the value, as we only check it from the enabled step.
This way we make sure there is no legacy string check anymore.

Reply:
I think we should test the actual function that is utilized by these features instead of each of its implementation, what do you think?

Reply:
Sure we can also do that, just meaning to say if you simply remove the `value: 'true'` we make sure to not introduce and fake false tests

Reply:
Makes total sense. Should be removed by now.